cmd/lit-af/shell.go

```diff
@@ -221,6 +221,12 @@ func (lc *litAfClient) Ls(textArgs []string) error {
 		return nil
 	}
 
+	if len(textArgs) == 0 {
+		fmt.Printf("pick one: conns, chans, dualfunds, txos, ports, addrs, bals")
```

Comment:
should either be `fmt.Println` or you should pass `%s` and then use `fmt.Printf`